pkgs/development/python-modules/libevdev/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "libevdev";
+  version = "0.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "10gwj08kn2rs4waq7807mq34cbavgkpg8fpir8mvnba601b8q4r4";
+  };
+
+  doCheck = false;
+
+  meta = with stdenv.lib; {
+    description = "Python wrapper around the libevdev C library";
+    homepage = "https://gitlab.freedesktop.org/libevdev/python-libevdev";
```

Comment:
`platforms` is missing.

Reply:
But this is clearly linux-specific, isn't it?

Reply:
Not necessarily, you can install X11 and `libev` on MacOS systems (apparently)

Nonetheless it probably still *builds* on those platforms even if it might not do anything.